Code/ObjectMapping/RKObjectMapping.m

```diff
@@ -69,8 +69,8 @@ - (RKObjectMapping *)invertMapping:(RKObjectMapping *)mapping withPredicate:(BOO
     inverseMapping = [RKObjectMapping requestMapping];
     (self.invertedMappings)[dictionaryKey] = inverseMapping;
     [inverseMapping copyPropertiesFromMapping:mapping];
-    // We want to serialize `nil` values
-    inverseMapping.assignsDefaultValueForMissingAttributes = YES;
+    // We potentially want to serialize `nil` values, adopt the strategy from the mapping we're inverting.
```

Comment:
Can you check the blame on that line & the comment and make sure it isn't that way for a reason?